IF/src/main/java/com/github/stefvanschie/inventoryframework/util/ItemUtil.java

```diff
@@ -0,0 +1,40 @@
+package com.github.stefvanschie.inventoryframework.util;
+
+import com.github.stefvanschie.inventoryframework.util.version.Version;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+
+public class ItemUtil {
+    private ItemUtil() {
+    }
+
+    @SuppressWarnings("deprecation")
+    public static ItemStack getPlayerHead() {
+        if (Version.CURRENT.getMinor() >= 13) {
+            return new ItemStack(Material.valueOf("PLAYER_HEAD"));
```

Comment:
Would be better to create fields for these ItemStack instances, so it will not create a new one one every call.